packages/common-ui-web/src/components/VehicleDynamicWireframe.tsx

```diff
@@ -0,0 +1,75 @@
+import { useMonkTheme } from '@monkvision/common';
```

Comment:
Put this component file in its own directory, with an `index.ts` file that exports the component. Also, the `index.ts` file should only export the necessary elements : the component, and the types declared, that's it. Take inspiration from the other components in the package please.

Also please do this for the other component